--- kio_gopher-20040214/admin/cvs.sh~	2007-12-30 18:33:44.456796095 +0200
+++ kio_gopher-20040214/admin/cvs.sh	2007-12-30 18:34:31.375000936 +0200
@@ -66,7 +66,7 @@
     echo "*** KDE requires automake $required_automake_version"
     exit 1
     ;;
-  automake*1.6.* | automake*1.7* | automake*1.8* ) : ;;
+  automake*1.6.* | automake*1.7* | automake*1.8* | automake*1.9* | automake*1.10* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** KDE requires automake $required_automake_version"
